docs/pages/reference/helm-reference/teleport-cluster.mdx

```diff
@@ -832,70 +945,54 @@ You can set `volumeSize` to request a different size of persistent volume when i
 
 ## `aws`
 
-| Can be used in `custom` mode? | `teleport.yaml` equivalent |
-| - | - |
-| ❌ | See [Using DynamoDB](../../reference/backends.mdx#dynamodb) and [Using Amazon S3](../../reference/backends.mdx#s3) for details |
-
 `aws` settings are described in the AWS guide: [Running an HA Teleport cluster using an AWS EKS Cluster](../../deploy-a-cluster/helm-deployments/aws.mdx)
 
 ## `gcp`
 
-| Can be used in `custom` mode? | `teleport.yaml` equivalent |
-| - | - |
-| ❌ | See [Using Firestore](../../reference/backends.mdx#dynamodb) and [Using GCS](../../reference/backends.mdx#gcs) for details |
-
 `gcp` settings are described in the GCP guide: [Running an HA Teleport cluster using a Google Cloud GKE cluster](../../deploy-a-cluster/helm-deployments/gcp.mdx)
 
-### `highAvailability`
+## `highAvailability`
 
-## `highAvailability.replicaCount`
+`highAvailability` contains settings controlling how Teleport pods are
+replicated and scheduled. This allows to run Teleport in a highly-available
+fashion: Teleport should sustain the crash/loss of a machine without interrupting
+the service.
 
-| Type | Default value | Can be used in `custom` mode? |
-| - | - | - |
-| `int` | `1` | ✅ (when using HA storage) |
-
-`highAvailability.replicaCount` can be used to set the number of replicas used in the deployment.
+**For auth pods**:
```

Comment:
Suggest using headers instead of using bold as a header:

```suggestion
### For auth pods
```

or 

```suggestion
#### For auth pods
```

Reply:
Unresolving this because the linter is not happy.

Using heading level 3 would make the section appear in the lateral toc which only contains value names, so this would be confusing.

Using header level 4 is OK, but the linter does not like that the heading level 3 is skipped.

Do you know a trick to make the linter happy with a level 4 header, or to hide a level 3 header from the toc?